### DISCUSSION TITLE
Possible to add classic guilds?

### DISCUSSION THREAD

**Discussion by waggz81** [Chalsean/XFaction#222] [Ideas]
With the bnet mesh you've developed here, would it also be possible to add classic guilds to this addon?

**Comment by Chalsean** [comment]
The way it currently is coded is that if someone does not have a BNet
friend online in the other faction, it will communicate with other guild
members via a custom channel.  Unfortunately Blizzard has this API disabled
for channel in classic:
https://wowpedia.fandom.com/wiki/API_C_ChatInfo.SendAddonMessage

In theory its possible to switch to whisper from channel but will require a
lot of rework.  Plus I'd have to double check all the other APIs to see
what else Blizz has disabled.  It's highly likely there are other API
differences that would need to be coded around.
